lib/absinthe/subscription.ex

```diff
@@ -109,7 +109,15 @@ defmodule Absinthe.Subscription do
   def subscribe(pubsub, field_key, doc_id, doc) do
     registry = pubsub |> registry_name
 
-    {:ok, _} = Registry.register(registry, field_key, {doc_id, doc})
+    doc_value = {
+      doc_id,
+      %{
+        initial_phases: doc.initial_phases,
```

Comment:
@bruce `:ets` does indeed flatten values. This turns out to be a problem for storing pipelines in `:ets`, because since we tend to do `{Phase, options}`, where `options` includes the context, the ~40 phases multiply the size of the context by ~40.

The simplest option here may be to rework the built in pipelines such that only the first phase takes the options, and it stores the relevant bits on the blueprint, and then later phases take no options.